doc/techref/projections.md

```diff
@@ -0,0 +1,9 @@
+# GMT Map Projections
+
+The table below shows the projection codes for the 31 GMT projections:
+
+| PyGMT Projection Argument | Projection Name |
```

Comment:
Refer to 
https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#subscript-superscript for subscript.